BP-Test/01_fresh_vegetable_competition
# Experiment：lightgbm

- Object
  - Test lightGBM for simple settings
  - Add files about introductions and docs about "mlflow"

- Data
  - Generated by [The output of an EDA](https://github.com/BP-Test/01_fresh_vegetable_competition/blob/add_lightgm_test/notebook/EDA.ipynb)
 
- Settings 
  - CV for 3 Folds.  (See details : [model_train.ipynb](https://github.com/BP-Test/01_fresh_vegetable_competition/blob/add_lightgm_test/notebook/model_train.ipynb)

- Result
  - bad

- Next Step
  1. Making a class for preprocessing and training
  2. Try mlflow workflow using  1. as introduced in [cb85d7c](https://github.com/BP-Test/01_fresh_vegetable_competition/pull/1/commits/cb85d7c42481d8d4454e819b4735e13549173bc3) (lower priority)
